packages/atomic/src/components/commerce/product-template-components/atomic-product-rating/atomic-product-rating.tsx

```diff
@@ -0,0 +1,133 @@
+import {Product, ProductTemplatesHelpers} from '@coveo/headless/commerce';
```

Comment:
This PR would have benefitted from a small overhaul/refactor of `atomic-result-rating` component, to extract some common pattern into functional components or shared utility function (similar to what was done past few weeks across many different files/components).

I don't think it would be a huge time sink, and would prevent adding more tech debt.